fastrand.go

```diff
@@ -111,6 +108,9 @@ func Intn(n int) int {
 }
 
 // BigIntn returns a uniform random value in [0,n). It panics if n <= 0.
+//
+// TODO: This particular function has not been extended to be faster, as it is
+// just a passthrough.
```

Comment:
Oh, I did not see that we are passing in our own reader. That's good enough then